chsdi/locale/fi/LC_MESSAGES/chsdi.po

```diff
@@ -2217,64 +2217,13 @@ msgid "ch.bafu.laerm-strassenlaerm_tag"
 msgstr "Canera da via, di"
 
 msgid "ch.bafu.landesforstinventar-baumarten"
-msgstr "Baumarten (LFI)"
+msgstr "Spezias da bostgs (IFN)"
 
 msgid "ch.bafu.landesforstinventar-totholz"
-msgstr "Totholz (LFI)"
+msgstr "Laina morta (IFN)"
 
 msgid "ch.bafu.landesforstinventar-waldanteil"
```

Comment:
Translations have been doubled. Translations are there so it is correcting a bad rebase. LGTM to me.